src/components/searchForm/searchForm.tsx

```diff
@@ -0,0 +1,19 @@
+import './searchForm.css';
+import { useState } from 'react';
+const SearchForm = ({initialSearchQuery, onSearch}: SearchFormProps) => {
+    const [searchText, setSearchText] = useState('');
+    const handleSearchTextChange = (e:React.ChangeEvent<HTMLInputElement>) => {
+        if(e.target){
+            setSearchText(e.target.value);
+        }
+    }
+return (
+    <>
+    <form className= "formContainer" onSubmit={onSearch}>
+        <input type= "text" className="searchInput" placeholder={initialSearchQuery} value={searchText} onChange={handleSearchTextChange} />
```

Comment:
placeholder={initialSearchQuery}
Placeholder text and the input value - are the different entities.